src/app/auth/shared/services/auth.service.ts

```diff
@@ -62,13 +67,8 @@ export class AuthService {
     return this.currentUser.asObservable();
   }
 
-  private makeHttpRequest(query) {
+  private makeHttpRequest(query): Observable<any> {
```

Comment:
My point was that a method `makeHttpRequest` is too abstract and the return type of `Observable<any>` just proves it.

Make the `url` the class property and use `httpClient` directly.